build_supported_mimetypes_table_rst.py

```diff
@@ -14,7 +14,7 @@
         print("+-{}-+-{}-+".format("".ljust(80, "-"), "".ljust(11, "-")))
 
         for mime in builder.get_supported_mimetypes():
-            ext = pm.get_file_extensions(mime) or " - "
+            ext = pm.get_file_extension(mime) or " - "
```

Comment:
fixing broken code ?

Reply:
Yup :] see:
```
$ git grep get_file_extension
build_supported_mimetypes_table_rst.py:            ext = pm.get_file_extension(mime) or " - "
preview_generator/manager.py:    def get_file_extension(self, mime: str) -> typing.Optional[str]:
$ 
```

It was spotted by `mypy` so I fixed it.

Reply:
:+1: